scribejava-core/src/main/java/com/github/scribejava/core/model/OAuth2ErrorResponse.java

```diff
@@ -0,0 +1,51 @@
+package com.github.scribejava.core.model;
+
+import com.github.scribejava.core.exceptions.OAuthException;
+
+import java.net.URI;
+
+/**
+ * Representing <a href="https://tools.ietf.org/html/rfc6749#section-5.2">"5.2. Error Response"</a>
+ */
+public class OAuth2ErrorResponse extends OAuthException {
```

Comment:
There are at least 2  Error Response in OAuth2 
second one is https://tools.ietf.org/html/rfc6749#section-7.2

I think it's better to name this one as OAuth2AccessTokenErrorResponse